Sources/NIOCore/TypeAssistedChannelHandler.swift

```diff
@@ -30,6 +30,11 @@ extension _EmittingChannelHandler {
     public func wrapOutboundOut(_ value: OutboundOut) -> NIOAny {
         return NIOAny(value)
     }
+
+    @inlinable
+    public static func wrapOutboundOut(_ value: OutboundOut) -> NIOAny {
```

Comment:
Can we think of a reason not to deprecate the non-static versions?

Reply:
I think it would be annoying to users having to change all of those :|.

What we should definitely remove though is the customisation point on those. There is only one correct implementation so I'd argue we could even remove those without a SemVer major break.

Reply:
While we can, I don't think it helps us much, and will cause mayhem for those few users who are using library evolution mode even though we told them not to.

It's definitely annoying to users, but otherwise the odds that anyone will get the perf win is pretty low. `self.` is going to be muscle memory for most, and already written for the rest.

Reply:
Fair